app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -383,6 +397,44 @@ export default {
     this.$root.$off('showSavingDialogEvent')
   },
   methods: {
+    updateBaseTitle() {
+      this.$nextTick(() =>
+        this.$nextTick(() => {
+          let title
+          const {
+            name: nameComponent,
+            params: {
+              tab: tabName
+            }
+          } = this.$route
+          if (nameComponent === "Dataset") {
+            if (this.titleDataset) {
+              const titleI18n = this.titleDataset
+                ? `${this.titleDataset} · `
+                : "";
+
+                if (tabName === 'editor') {
+                  const tabTitle = `${this.labelData} · `
+                  title = `${titleI18n} ${tabTitle} ${this.pageTitle}`;
```

Comment:
this expression is repeated in all if statements, could it be done outside the ifs, right?

Reply:
Definitely. `let tabTible` in each clause and join them all at the end